tls-tester/src/com/axonivy/test/tls/TLSTest.java

```diff
@@ -172,16 +171,10 @@ private void loadCustomClientKeystore()
   private void loadCustomClientTruststore()
   {
     TLSTestData data = new TLSTestData(TLSTestGroup.CLIENT_CUSTOM_TRUSTSTORE);
-    if (!sslClientSettings.useCustomTrustStore())
-    {
-      data.addEntry("Custom client TrustStore set to NOT be used - skipping");
-      data.setResult(2);
-    } else
-    {
-      customTrustStore = loadKeyStore(sslClientSettings.getTrustStoreFile(), sslClientSettings.getTrustStorePassword()
-          .toCharArray(), sslClientSettings.getTrustStoreType(), sslClientSettings.getTrustStoreProvider(),
-          "CustomTrustStore", data);
-    }
```

Comment:
don't remove this feature right now; it's still valid if you upload your own keystore on the tls-tester dialog

Reply:
The problem is that we no longer have the useCustomTrustStore method.

Reply:
ahh; you are right ... good then.